netbox/netbox/settings.py

```diff
@@ -156,6 +156,7 @@
 STORAGE_BACKEND = getattr(configuration, 'STORAGE_BACKEND', None)
 STORAGE_CONFIG = getattr(configuration, 'STORAGE_CONFIG', {})
 TIME_ZONE = getattr(configuration, 'TIME_ZONE', 'UTC')
+USE_I18N = getattr(configuration, 'USE_I18N', True)
```

Comment:
While an argument can be made for preserving Django setting names, I expect the term i18n won't make sense to most users. IMO we should call this something like `ENABLE_TRANSLATION`.